india_compliance/gst_india/report/gst_sales_register/gst_sales_register.py

```diff
@@ -19,6 +19,14 @@ def execute(filters=None):
 
 def validate_filters(filters):
     filters = frappe._dict(filters)
+
+    if filters.invoice_sub_category:
```

Comment:
This is incorrect fix. Will fail for existing cases that work.

<img width="1237" height="477" alt="Image" src="https://github.com/user-attachments/assets/97b50551-0c08-430d-ba36-fa9b916c8bc1" />